### PR TITLE
🔀[mH] Don't show at Krankentransporte

### PR DESCRIPTION
**Pull request by Cr4zyc4k3**
**What kind of update does this PR provide?** *Please check*

- [ ] new translations / updated translations / translation fixes
- [x] a new feature
- [ ] a new module
- [ ] a bugfix for an existing feature / module
- [ ] improvement of style or user experience
- [ ] other: *Please fill out*

**What is included in your update?**
* #25 
> hide for Krankentransporte

**Additional notes**
/

### CODE REVIEW

**Comment by Cr4zyc4k3** [comment]
@LSS-Manager deploy

**Comment by Cr4zyc4k3** [comment]
@LSS-Manager deploy

**Comment by Cr4zyc4k3** [comment]
@LSS-Manager deploy

**Comment by Cr4zyc4k3** [comment]
@LSS-Manager deploy

**Comment by Cr4zyc4k3** [comment]
@LSS-Manager deploy

**Comment by Cr4zyc4k3** [comment]
@LSS-Manager deploy

File: src/modules/missionHelper/i18n/de_DE.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "expansions": "Ausbreitung | Ausbreitungen",
     "followups": "Folgeeinsatz | Folgeeinsätze",
     "generated_by": "Generiert von",
+    "ignore_mission_IDs": [143,613],
```
Comment (jxn-30) [inline src/modules/missionHelper/i18n/de_DE.json:6]: we could use the root `transfer_missions` translation instead. That would reduce redundant data.

File: src/modules/missionHelper/i18n/cs_CZ.root.json [hunk 2]
```diff
@@ -15,6 +15,10 @@
         "title": "Special requirements"
     },
     "settings": {
+        "hide_on_Krankentransport": {
+            "description": "Hide on ambulance missions",
+            "title": "Hide on ambulance missions"
```
Comment (jxn-30) [inline src/modules/missionHelper/i18n/cs_CZ.root.json:20]: I would suggest/prefer "patient transfer missions". (Same for other EN translations)

File: src/modules/missionHelper/i18n/es_ES.root.json [hunk 3]
```diff
@@ -20,6 +20,10 @@
         }
     },
     "settings": {
+        "hide_on_Krankentransport": {
+            "description": "Verstecke den Einsatzhelfer bei Kranken/Intensivtransporten",
+            "title": "Verstecke bei Krankentransport"
+        },
```
Comment (jxn-30) [inline src/modules/missionHelper/i18n/es_ES.root.json:26]: can we have at least EN translation on non-DE countries?

**Review by jxn-30 — CHANGES_REQUESTED** [review]

**Comment by jxn-30** [comment]
@LSS-Manager deploy

File: src/modules/missionHelper/main.ts [hunk 4]
```diff
@@ -8,6 +8,15 @@ export default (async ({ LSSM, $m, $mc, getSetting, setSetting }) => {
...
         return;
+    getSetting('hide_on_Krankentransport').then(hideBool => {
+        const missionID = parseInt(LSSM.$utils.getMissionTypeInMissionWindow());
+
+        if (
+            Object.values(LSSM.$t('transfer_missions')).includes(missionID) &&
+            hideBool
+        )
+            return false;
+    });
```
Comment (jxn-30) [inline src/modules/missionHelper/main.ts:19]: this does not work as `return` onyl exits the promise but not the modules function.

I would suggest the following version (untested):
```suggestion
    const doNotLoad = await getSetting('hide_on_Krankentransport').then(hideBool => {
        const missionID = parseInt(LSSM.$utils.getMissionTypeInMissionWindow());

        return
            Object.values(LSSM.$t('transfer_missions')).includes(missionID) &&
            hideBool;
    });
    if (doNotLoad) return;
```

**Review by jxn-30 — CHANGES_REQUESTED** [review]

**Comment by jxn-30** [comment]
@LSS-Manager deploy

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**